pallets/metaverse/src/lib.rs

```diff
@@ -718,26 +718,35 @@ impl<T: Config> Pallet<T> {
 
 	pub fn upgrade_metaverse_info_v2() -> Weight {
 		log::info!("Start upgrade_metaverse_info_v2");
-		let mut num_metaverse_items = 0;
+		let mut upgraded_metaverse_items = 0;
+		let mut total_metaverse_items = 0;
+
+		Metaverses::<T>::translate(|k, metaverse_info_v1: MetaverseInfoV1<T::AccountId>| {
```

Comment:
All Metaverse struct should have landClassId & estateClassId on development chain. We should only generate new landClassId & estateClassId for the structs have landClassId: 0  & estateClassId: 1  as default value.


This will avoid override the default value for metaverse created after the last runtime upgrade.

Reply:
sure I've just added  storage v3 migration and changed the v2 migration to set a default landClassId: 0 & estateClassId: 1